playground/frontend/lib/modules/examples/repositories/models/shared_file_model.dart

```diff
@@ -16,12 +16,14 @@
  * limitations under the License.
  */
 
-class SelectorPositionModel {
-  final double xAlignment;
-  final double yAlignment;
+class SharedFile {
+  final String code;
+  final bool isMain;
+  final String name;
 
-  const SelectorPositionModel({
-    required this.xAlignment,
-    required this.yAlignment,
+  const SharedFile({
+    required this.code,
+    this.name = '',
+    this.isMain = false,
```

Comment:
Second thoughts, `isMain` is better be required. Using empty name by default seems obvious to the reader, but they will be puzzled to guess what is `isMain` if it is missed. I do not see a reason to prefer `true` or `false` here.